### PR TITLE
Update XR Index page

### PR DESCRIPTION
**Pull request by skyace65**
Removes a note which Bastiaan confirmed is no longer needed in rocket chat. Also adds a brief description to the page.

### CODE REVIEW

**Review by mhilbrunner — APPROVED** [review]

File: tutorials/xr/index.rst [hunk 1]
```diff
@@ -3,6 +3,9 @@
 XR
 ==
 
+This section of the manual covers everything related to XR(
```
Comment (mhilbrunner) [inline tutorials/xr/index.rst:6]:
```suggestion
This section of the manual covers everything related to XR (
```

**Comment by skyace65** [comment]
Fixed.

**Review by Piralein — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thanks for the update and your work! Merged.

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2 in https://github.com/godotengine/godot-docs/pull/9648.

**Pull request merged**